api/spec/controllers/spree/api/option_types_controller_spec.rb

```diff
@@ -35,7 +35,7 @@ def check_option_values(option_values)
 
     it "can retrieve a list of specific option types" do
       option_type_1 = create(:option_type)
-      option_type_2 = create(:option_type)
+      create(:option_type)
```

Comment:
If we're here, why not follow "given, when, then" and put this into a before block? `option_type1` could even be a `let!`